src/model/beneficial.owner.type.model.ts

```diff
@@ -1,4 +1,5 @@
 export const BeneficialOwnerTypeKey = "beneficial_owner_type";
+// export const ManagingOfficerTypeKey = "managing_officer_type";
 
```

Comment:
Not sure if a separate variable should be made for MO type, currently works but may have to change when both add MO pages have been made

Reply:
Same as above I think this will be picked up as a code smell if left on its own

Reply:
Fixed